packages/subgraph-service/contracts/interfaces/ISubgraphService.sol

```diff
@@ -112,36 +112,43 @@ interface ISubgraphService is IDataServiceFees {
     /**
      * @notice Thrown when trying to force close an allocation that is not stale
      * @param allocationId The id of the allocation
-     * @param lastPOIPresentedAt The timestamp when the last POI was presented
      */
-    error SubgraphServiceAllocationNotStale(address allocationId, uint256 lastPOIPresentedAt);
+    error SubgraphServiceCannotForceCloseAllocation(address allocationId);
```

Comment:
Here we need to update the comments to explain both cases where the error could be thrown (stale and overallocated)